From 0a62a6db2770d51eaee2a3cfc17046a4d7c26476 Mon Sep 17 00:00:00 2001
From: Rael Gugelmin Cunha <rael.gc@gmail.com>
Date: Mon, 5 Nov 2018 13:42:36 -0200
Subject: [PATCH] Removing no more used mameNameToRealName

---
 es-app/src/PlatformId.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/es-app/src/PlatformId.cpp b/es-app/src/PlatformId.cpp
index 9abbf8b4c..fa9af434a 100644
--- a/es-app/src/PlatformId.cpp
+++ b/es-app/src/PlatformId.cpp
@@ -2,8 +2,6 @@
 
 #include <string.h>
 
-extern const char* mameNameToRealName[];
-
 namespace PlatformIds
 {
 	const char* PlatformNames[PLATFORM_COUNT + 1] = {
