From 15664c1a9055334b6845a8780051e8f49f4b2f7a Mon Sep 17 00:00:00 2001
From: derselbst <tom.mbrt@googlemail.com>
Date: Sat, 12 Jan 2019 08:14:51 +0100
Subject: [PATCH] another cross compilation fix

fixes #501
---
 src/gentables/CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/gentables/CMakeLists.txt b/src/gentables/CMakeLists.txt
index 64ec7b44..638f299c 100644
--- a/src/gentables/CMakeLists.txt
+++ b/src/gentables/CMakeLists.txt
@@ -4,6 +4,12 @@ cmake_minimum_required(VERSION 3.1)
 # which hopefully will be the host compiler
 unset(ENV{CC})
 
+# also unset $CFLAGS to avoid passing any cross compilation flags to the host compiler
+unset(ENV{CFLAGS})
+
+# linker flags as well
+unset(ENV{LDFLAGS})
+
 project (gentables C)
 
 set ( CMAKE_BUILD_TYPE Debug )
