--- a/src/themes/themes.qrc
+++ b/src/themes/themes.qrc
@@ -17,6 +17,7 @@
         <file>pegasus-theme-grid/layer_platform/PlatformCard.qml</file>
         <file>pegasus-theme-grid/assets/icons/heart_empty.svg</file>
         <file>pegasus-theme-grid/assets/icons/heart_filled.svg</file>
+        <file>pegasus-theme-grid/assets/logos/2048.svg</file>
         <file>pegasus-theme-grid/assets/logos/3do.svg</file>
         <file>pegasus-theme-grid/assets/logos/amiga.svg</file>
         <file>pegasus-theme-grid/assets/logos/amstradcpc.svg</file>
@@ -32,6 +33,7 @@
         <file>pegasus-theme-grid/assets/logos/atarilynx.svg</file>
         <file>pegasus-theme-grid/assets/logos/atarist.svg</file>
         <file>pegasus-theme-grid/assets/logos/c64.svg</file>
+        <file>pegasus-theme-grid/assets/logos/chailove.svg</file>
         <file>pegasus-theme-grid/assets/logos/coco.svg</file>
         <file>pegasus-theme-grid/assets/logos/colecovision.svg</file>
         <file>pegasus-theme-grid/assets/logos/daphne.svg</file>
@@ -41,19 +43,25 @@
         <file>pegasus-theme-grid/assets/logos/fds.svg</file>
         <file>pegasus-theme-grid/assets/logos/gamegear.svg</file>
         <file>pegasus-theme-grid/assets/logos/gb.svg</file>
+        <file>pegasus-theme-grid/assets/logos/gc.svg</file>
         <file>pegasus-theme-grid/assets/logos/gba.svg</file>
         <file>pegasus-theme-grid/assets/logos/gbc.svg</file>
         <file>pegasus-theme-grid/assets/logos/genesis.svg</file>
         <file>pegasus-theme-grid/assets/logos/gog.svg</file>
         <file>pegasus-theme-grid/assets/logos/intellivision.svg</file>
         <file>pegasus-theme-grid/assets/logos/kodi.svg</file>
+        <file>pegasus-theme-grid/assets/logos/linux-games.svg</file>
         <file>pegasus-theme-grid/assets/logos/love.svg</file>
         <file>pegasus-theme-grid/assets/logos/macintosh.svg</file>
         <file>pegasus-theme-grid/assets/logos/mame.svg</file>
         <file>pegasus-theme-grid/assets/logos/mastersystem.svg</file>
         <file>pegasus-theme-grid/assets/logos/megadrive.svg</file>
+        <file>pegasus-theme-grid/assets/logos/moonlight.svg</file>
+        <file>pegasus-theme-grid/assets/logos/mrboom.svg</file>
         <file>pegasus-theme-grid/assets/logos/msx.svg</file>
+        <file>pegasus-theme-grid/assets/logos/n3ds.svg</file>
         <file>pegasus-theme-grid/assets/logos/n64.svg</file>
+        <file>pegasus-theme-grid/assets/logos/nds.svg</file>
         <file>pegasus-theme-grid/assets/logos/neogeo.svg</file>
         <file>pegasus-theme-grid/assets/logos/neogeocd.svg</file>
         <file>pegasus-theme-grid/assets/logos/nes.svg</file>
@@ -61,12 +69,16 @@
         <file>pegasus-theme-grid/assets/logos/ngpc.svg</file>
         <file>pegasus-theme-grid/assets/logos/odyssey2.svg</file>
         <file>pegasus-theme-grid/assets/logos/pc.svg</file>
+        <file>pegasus-theme-grid/assets/logos/pce-cd.svg</file>
         <file>pegasus-theme-grid/assets/logos/pcengine.svg</file>
         <file>pegasus-theme-grid/assets/logos/ports.svg</file>
+        <file>pegasus-theme-grid/assets/logos/prboom.svg</file>
         <file>pegasus-theme-grid/assets/logos/psp.svg</file>
         <file>pegasus-theme-grid/assets/logos/psx.svg</file>
+        <file>pegasus-theme-grid/assets/logos/ps2.svg</file>
         <file>pegasus-theme-grid/assets/logos/residualvm.svg</file>
         <file>pegasus-theme-grid/assets/logos/retropie.svg</file>
+        <file>pegasus-theme-grid/assets/logos/saturn.svg</file>
         <file>pegasus-theme-grid/assets/logos/scummvm.svg</file>
         <file>pegasus-theme-grid/assets/logos/sega32x.svg</file>
         <file>pegasus-theme-grid/assets/logos/segacd.svg</file>
@@ -74,9 +86,11 @@
         <file>pegasus-theme-grid/assets/logos/snes.svg</file>
         <file>pegasus-theme-grid/assets/logos/steam.svg</file>
         <file>pegasus-theme-grid/assets/logos/turbografx16.svg</file>
+        <file>pegasus-theme-grid/assets/logos/tyrquake.svg</file>
         <file>pegasus-theme-grid/assets/logos/vectrex.svg</file>
         <file>pegasus-theme-grid/assets/logos/videopac.svg</file>
         <file>pegasus-theme-grid/assets/logos/virtualboy.svg</file>
+        <file>pegasus-theme-grid/assets/logos/wii.svg</file>
         <file>pegasus-theme-grid/assets/logos/wonderswan.svg</file>
         <file>pegasus-theme-grid/assets/logos/wonderswancolor.svg</file>
         <file>pegasus-theme-grid/assets/logos/zmachine.svg</file>

