--- a/src/Graphics/OpenGLContext/GLFunctions.cpp
+++ b/src/Graphics/OpenGLContext/GLFunctions.cpp
@@ -199,7 +199,7 @@
 void initGLFunctions()
 {
 #ifdef VC
-	void *gles2so = dlopen("/opt/vc/lib/libbrcmGLESv2.so", RTLD_NOW);
+	void *gles2so = dlopen("/usr/lib/libbrcmGLESv2.so", RTLD_NOW);
 #elif defined(ODROID)
 	void *gles2so = dlopen("/usr/lib/arm-linux-gnueabihf/libGLESv2.so", RTLD_NOW);
 #elif defined(VERO4K)

